sanctuary/sanctuary-types.d.ts

```diff
@@ -54,6 +54,7 @@ type NonZeroFiniteNumber = number; // - Infinity, -Infinity, 0, -0, NaN
 type ValidNumber = number; // - NaN
 
 export interface Functor<F> {
+  map<U>(fn: (f: F) => U): Functor<U>;
```

Comment:
Yes, it is. See [prefixed method names][1].


[1]: https://github.com/fantasyland/fantasy-land/tree/v3.1.0#prefixed-method-names